docs(keycloak): add manage group via claims (#2739)
kestra-io/docs: content/docs/06.enterprise/03.auth/sso/keycloak.md

2. Click on the **roles** scope
3. Select **Mappers**
4. Click on **client roles**
5. Enable **Add to ID Token**

In case the interface changes to Keycloak, check out their documentation for [managing resources and scopes](https://www.keycloak.org/docs/latest/authorization_services/#_resource_overview).

### Configure Kestra

After adding or updating your roles, configure Kestra to fetch the `roles` scope like in following Micronaut configuration; make note of the added `scopes` parameter where the information must be added:

```yaml
micronaut:
  security:
    oauth2:
      enabled: true
      clients:
        keycloak:
          client-id: "kestra"
          client-secret: "my-secret"
          openid:
            issuer: "http://localhost:8088/realms/kestra"
          scopes: ["openid", "profile", "email", "roles"] # <-- needs to be added
      endpoints:
        logout:
          get-allowed: true
```

With this configured, you then need to update the `kestra` property in your configuration file to synchronize groups from this claim with the `groups-claim-path`:

```yaml
kestra:
  security:    
    oidc:
      groups-claim-path: "resource_access.kestra.roles"
```

> Once the synchronization connection is made, you are now able to use oauth claims to source user roles with Keycloak as the single source of truth

Reviewer: ```suggestion
Once the synchronization connection is made, you can use OAuth claims to source user roles with Keycloak as the single source of truth.
```
